list.json

```diff
@@ -870,5 +870,14 @@
   "zgrab",
   "Zippy",
   "ZnajdzFoto/Image",
-  "ZyBorg"
+  "ZyBorg",
+  "WhatsApp",
+  "Twitterbot/1.0",
+  "LinkedInBot",
+  "Quora",
+  "Baidu",
+  "Applebot 0.1",
+  "TelegramBot",
+  "Slack",
+  "Snapchat",
```

Comment:
Could you specify which user agent string you wich to check here? I don't think the in-app browser (`Mozilla/5.0 (iPhone; CPU iPhone OS 13_3_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.5 Mobile/15E148 Snapchat/10.77.5.59 (like Safari/604.1)`) should be considered a bot

Reply:
You know the social media apps crawl the meta-tags(Open Graph) on a website when a user posted any links, therefore, I've added these links.